File: Baekjoon/2920_sol1.cpp
// https://www.acmicpc.net/problem/2920
// Solution 1
// c++

# include <iostream>
using namespace std;

int main() {
    int num[8] = {0, };

    for(int i=0 ; i<8 ; i++) {
        cin >> num[i];
    }
    
    bool asc = true; 
    bool dsc = true; 

    for(int i=1 ; i<8 ; i++) {
        if (num[i] < num[i-1]) 
            asc = false;
        if (num[i-1] < num[i]) 
            dsc = false;
    }

    if (asc == true) 
        cout<<"ascending\n";
    else if (dsc == true)
        cout<<"descending\n";
    else
        cout<<"mixed\n";
    return 0;
}

File: Baekjoon/2920_sol1.py
# https://www.acmicpc.net/problem/2920
# Solution 1
# Python

num = input()
num_list = num.split(' ')

isAscending = True # Assume the numbers are in an ascending order

# Check if the numbers are in an ascending order
for i in range(1, len(num_list)) :
    if num_list[i] < num_list[i-1] : 
        isAscending = False

isDescending = True # Assume the numbers are in an descending order

# Check if the numbers are in a descending order
for i in range(1, len(num_list)) :
    if num_list[i-1] < num_list[i] :
        isDescending = False

if isAscending == True :
    print("ascending")
elif isDescending == True :
    print("descending")
else :
    print("mixed")